packages/contracts/contracts/core/component/DaoAuthorizableCloneable.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+import {_auth} from "../../utils/auth.sol";
+import {IDAO} from "../IDAO.sol";
+
+/// @title DaoAuthorizableCloneable
+/// @author Aragon Association - 2022
+/// @notice An abstract contract providing a meta-transaction compatible modifier for clonable contracts to authorize function calls through an associated DAO.
+/// @dev Make sure to call `__DaoAuthorizableCloneable_init` during initialization of the inheriting contract.
+///      This contract is compatible with meta transactions through OZ's `Context`.
+abstract contract DaoAuthorizableCloneable is Initializable, Context {
```

Comment:
Personally I would use another abstraction for `Initializable` like `DaoAuthorizableInitializable` and remove the Initializable and duplicated code from `DaoAuthorizableCloneable` and `DaoAuthorizableUpgradeable`

Reply:
Good remark. Changed in https://github.com/aragon/core/pull/105/commits/9ed1dfc3a1061e74820ff72ac37b5ecef30ded9a.